code-challenges.js

```diff
@@ -30,8 +63,25 @@ var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "tem
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
+//declare a function that takes in an array
+const onlyNumbers = (array) =>{
+    //create a new variable and use map to get every value that is a number using typeof
+    let getValues = array.map ((value) =>{
+        //if it is a number, return the number
+        if(typeof value === "number"){
+            return value
+        }
+    })
+    //sort the numbers from least to greatest
+    let numSort = getValues.sort((a, b) => a - b)
+    //filter out the undefined values
+    let noUndefined = numSort.filter(value => value !== undefined)
```

Comment:
Very clever way to get those undefined out of there! 

I think a way we could refactor this, or keep in mind in the future is the difference between .map and .filter. You first used .map to kind of.. filter through the array to get the numbers, which returned a same length array with only numbers. You could just use filter right off the bat, which would accomplish what you're looking to do! :)